### PR TITLE
Replace pdf parsing libs

### PR DESCRIPTION
**Pull request by dagardner-nv**
## Description
* Use pypdfium2 for PDF parsing
* Includes CI fix from PR #1860 

## By Submitting this PR I confirm:
- I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
- When the PR is ready for review, new or existing tests cover these changes.
- When the PR is ready for review, the documentation is up to date with these changes.

### CODE REVIEW

File: examples/llm/vdb_upload/module/content_extractor_module.py [hunk 1]
```diff
@@ -22,7 +22,6 @@
 from typing import Dict
```
Comment (drobison00) [inline examples/llm/vdb_upload/module/content_extractor_module.py:22]:
```suggestion
from typing import Dict
import pdfium
```

File: examples/llm/vdb_upload/module/content_extractor_module.py [hunk 2]
```diff
@@ -171,12 +170,7 @@ def wrapper(input_info: ConverterInputInfo, *args, **kwargs):
 
 @_converter_error_handler
 def _pdf_to_text_converter(input_info: ConverterInputInfo) -> str:
-    text = ""
-    pdf_document = fitz.open(stream=input_info.io_bytes, filetype="pdf")
-    for page_num in range(pdf_document.page_count):
-        page = pdf_document[page_num]
-        text += page.get_text()
-    return text
+    raise NotImplementedError("PDF to text conversion is not implemented.")
```
Comment (drobison00) [inline examples/llm/vdb_upload/module/content_extractor_module.py:173]:
```suggestion
    text = ""
    pdf_document = libpdfium.PdfDocument(pdf_stream)
    for page_idx in range(len(pdf_document)):
        page = doc.get_page(page_idx)
        textpage = page.get_textpage()
        page_text = textpage.get_text_bounded()
        text += page_text
        
    return text
```

Comment (drobison00) [reply examples/llm/vdb_upload/module/content_extractor_module.py:173]: We moved NV-ingest over to use PDFium, its an apache-2 license.

**Comment by AnuradhaKaruppiah** [comment]
Nice! you turned this into an easy swap out.

**Review by AnuradhaKaruppiah — APPROVED** [review]

**Review by drobison00 — APPROVED** [review]

**Comment by BartleyR** [comment]
Reviewed and approved

**Review by BartleyR — APPROVED** [review]

**Pull request merged**